packages/xrpl/src/client/index.ts

```diff
@@ -938,8 +938,8 @@ class Client extends EventEmitter<EventTypes> {
         const accountLinesBalance = linesResponses.flatMap((response) =>
           formatBalances(response.result.lines),
         )
-        if (xrpBalance !== '') {
-          balances.push({ currency: 'XRP', value: xrpBalance })
+        if (xrpBalance !== 0) {
+          balances.push({ currency: 'XRP', value: xrpBalance.toString() })
```

Comment:
This might be a broader question than this PR, but perhaps there should be an error thrown if the XRP balance is somehow 0, since that shouldn't be possible?